IMC_WebDev/#Driving_the_car_and_flowing_the_water_with_CSS/blog.md

```diff
@@ -133,4 +134,4 @@ Similar for the car, but we want the car to go onto the bridge, so we break its
     ```
     
 ## Conclusion
```

Comment:
Would add more to this conclusion, perhaps expanding on/ suggesting other ways animation can be used for the mini city or other projects. Also, would choose another word choice besides "strong" because it's a bit vague; do you mean something more like "exciting," "impressive," "lively"?